lib/Site.js

```diff
@@ -275,6 +280,22 @@ Site.prototype.buildSourceFiles = function () {
   });
 };
 
+/**
+ * Build data index of the site
+ */
+Site.prototype.writeSiteData = function () {
```

Comment:
Move to after `Site.prototype.copyMarkBindAsset`.

Reply:
Move to after `Site.prototype.copyMarkBindAsset`.

Reply:
This has been changed